### DISCUSSION TITLE
CADViewer tie-in

### DISCUSSION THREAD

**Discussion by foxhoundv** [nextcloud/all-in-one#1931] [Ideas] [answered]
I was having an issue with a new app which wasn't opening files properly and their recommendation was:

"our developer feedback for the reference installation of Nextcloud-AIO, is for you to add: RewriteCond %{REQUEST_FILENAME} !/apps/cadviewer/converter/php/*\.*, into .htaccess in root of installation /var/www/html ,"

The link to the discussion is here... [CADViewer](https://github.com/CADViewer/NextCloud/issues/3)

**Comment by szaimen** [comment, marked as answer]
Hi, basically this doesnt look like the right way to make the App working, imho.

AIO doesn't do anything unusual regarding its webserver config, so if it doesn't work on AIO, it will most likely also not work on other Nextcloud instances.

However if they really require this in order to make the app work correctly, you could install AIO behind a reverse proxy by following [this documentation](https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md) at any time and add the required config directly to the reverse proxy config.

**Reply by szaimen** [reply]
Another approach would be to ask them to create a pr here: https://github.com/nextcloud/server/blob/master/.htaccess in order to add their change to the official htaccess but I doubt that such a PR would be accepted.

**Reply by foxhoundv** [reply]
> Hi, basically this doesnt look like the right way to make the App working, imho.
> 
> AIO doesn't do anything unusual regarding its webserver config, so if it doesn't work on AIO, it will most likely also not work on other Nextcloud instances.
> 
> However if they really require this in order to make the app work correctly, you could install AIO behind a reverse proxy by following [this documentation](https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md) at any time and add the required config directly to the reverse proxy config.

Yeah, I do have mine behind a reverse proxy, I am just so new at it I am still getting my head wrapped around the configuration side. PFSense + HAProxy. Thanks for helping to point in the right direction.
